eventstore/dynamodb/eventstore.go

```diff
@@ -65,14 +69,26 @@ func (c *EventStoreConfig) provideDefaults() {
 	if c.Region == "" {
 		c.Region = "us-east-1"
 	}
+	if c.Endpoint == "" {
+		c.Endpoint = "http://localhost:8000"
+	}
+	if c.Username == "" {
+		c.Username = "doesnotmatter"
+	}
+	if c.Password == "" {
+		c.Password = "doesnotmatter"
+	}
 }
 
 // NewEventStore creates a new EventStore.
 func NewEventStore(config *EventStoreConfig) (*EventStore, error) {
 	config.provideDefaults()
 
+	creds := credentials.NewStaticCredentials(config.Username, config.Password, "")
```

Comment:
Hmm, it should not fail if you have setup your AWS CLI and are able to use that. CI for the project is already setup and passing all tests, see badge at the front page. I will definitely try the local DynamoDB container as an alternative, as commented below.